prefetchLanguages.nix

```diff
@@ -0,0 +1,114 @@
+{ pkgs, ... }:
+
+let
+  inherit (builtins)
+    attrNames
+    concatStringsSep
+    mapAttrs
+    toFile
+    readFile
+    toJSON
+    fromJSON
+    baseNameOf
+    ;
+
+  inherit (pkgs) fetchgit nickel;
+  inherit (pkgs.lib) warn;
+  inherit (pkgs.stdenvNoCC) mkDerivation;
+  inherit (pkgs.tree-sitter) buildGrammar;
```

Comment:
Nitpick: it's quite a matter of personal taste, but it seems `buildGrammar` (and may other inherited symbols as well) is used only once in this whole file. In this case I'm a bit skeptical of the benefit of introducing an unqualified symbol (when I read `buildGrammar` for the first time below, it's a little bit less clear if it's coming from Nix or from this file, as opposed to a fully qualified `pkgs.tree-sitter.buildGrammar` - when you use it many times, there's of course a trade-off, but if it's only used once...). Anyway, also feel free to ignore :slightly_smiling_face:

Reply:
Yeah, I wondered about that while writing it, actually. Fixed in 8429690dd3393c3ca5ef2909ee3a462fa3377b77.